### ISSUE TITLE
Add message on Server Join

### ISSUE THREAD

**Issue by synsypa** [zhangchuck/discord_quote_bot#11]
tried to add a message on server join, but didn't work. Should be useful when the bot goes down to indicate revival

```
@bot.event
@asyncio.coroutine
def on_server_join():
    log.info(log_msg(['bot join', bot.user.name, bot.user.id, 'server', bot.server.id]))
            
    yield from bot.say('yo, we in there')

    log.info(log_msg(['sent_message', 'server_join', ctx.message.channel.name]))
```
was my attempt

**Comment by zhangchuck** [comment]
https://stackoverflow.com/questions/49881606/how-do-i-send-a-message-to-all-text-channels-with-discord-py

**Comment by zhangchuck** [comment]
Oddly, can't seem to get the `on_server_join()` event to trigger, even when turning it on in the server settings.

I got it to work by moving it to the `on_ready()` event.

**Comment by zhangchuck** [comment]
Addressed in #35
